sunkit_image/flct/flct.py

```diff
@@ -0,0 +1,519 @@
+    array2 : `numpy.ndarray`
+        The second array whose order is to be changed.
+
+    Returns
+    -------
+    `tuple`
+        It returns the two input arrays after changing their order from column major to
+        row major.
+    """
+
+    # Checking whether the C extension is correctly built.
+    if _pyflct is None:
+        raise ImportError("C extension for flct is missing, please rebuild.")
+
+    one, two = _pyflct.swap_order_two(array1, array2)
+
+    return (one, two)
+
+
+def column_row_of_three(array1, array2, array3):
+    """
+    Takes three arrays and swaps the order in which they were stored i.e.
+    changing from column major to row major and **not** the vice-versa. This
+    may change the values stored in the array as the arrays are first converted
+    to a binary format and then the order change takes place.
+
+    Parameters
+    ----------
+    array1 : `numpy.ndarray`
+        The first array whose order is to be changed.
+    array2 : `numpy.ndarray`
+        The second array whose order is to be changed.
+    array3 : `numpy.ndarray`
+        The third array whose order is to be changed.
+
+    Returns
+    -------
+    `tuple`
+        It returns the two input arrays after changing their order from column major to
+        row major.
+    """
+
+    # Checking whether the C extension is correctly built.
+    if _pyflct is None:
+        raise ImportError("C extension for flct is missing, please rebuild.")
+
+    one, two, three = _pyflct.swap_order_three(array1, array2, array3)
+
+    return (one, two, three)
+
+
+def flct(
+    image1,
+    image2,
+    deltat,
+    deltas,
+    sigma,
+    order="row",
+    quiet=False,
+    biascor=False,
+    thresh=0.0,
+    absflag=False,
+    skip=None,
+    xoff=0,
+    yoff=0,
+    interp=False,
+    kr=None,
+    pc=False,
+    latmin=0,
+    latmax=0.2,
+):
+    """
+    Performs Fourier Local Correlation Tracking by calling the FLCT C library.
+
+    .. note::
+
+        * In the references there are some dat files which can be used to test the FLCT code. The
+          best method to read those dat files is the `sunkit_image.flct.read_2_images` and
+          `sunkit_image.flct.read_3_images` as the arrays would automatically be read in row major
+          format.
+        * If you use the IDL IO routines to get the input arrays from ``dat`` files,
+          the IDL routines always read the binary files in the column major, but both Python and C,
+          on which these functions are written are row major so the order of the arrays have to be
+          changed which can be done with the ``order`` keyword. This may lead to different values
+          in both the cases.
+        * If your input arrays are column major then pass the `order` parameter as `column` and it
+          will automatically take care of the order change. But this can produce some changes in
+          the values of the arrays.
+
+    .. note::
```

Comment:
Do we need to do this twice?

Reply:
I just did this because both the notes belong to different themes. I mean the first set is only applicable to python wrapper which was introduced by us but the next set is the drawbacks of the algo itself which is both in C and python versions

Reply:
I would make then this block a warning over a note.